allennlp/training/trainer.py

```diff
@@ -106,6 +113,7 @@ def train(self) -> None:
                 if self._grad_norm:
                     clip_grad_norm(self._model.parameters(), self._grad_norm)
                 self._optimizer.step()
+            metrics = self._model.get_metrics(reset=True) or {"loss": train_loss}
```

Comment:
I don't think this is right.  You don't want metrics _or_ loss, you want metrics (if any) _and_ loss.  The loss is just another metric, one that's _always_ computed and added to whatever else is already there.